resources/views/user/alfonso-language-exams.blade.php

```diff
@@ -1,22 +1,44 @@
 <div class="row">
     <div class="col s12">
         <h6>Feltöltött nyelvvizsgák:</h6>
-        <blockquote>
-            <ul>
-            @forelse ($user->educationalInformation?->languageExams?->sortBy('date') ?? [] as $exam)
-                <li>
-                    <a target="_blank" href="/{{ $exam->path }}">
-                        {{ __('role.'.$exam->language) }} - {{ $exam->level }}</a>
-                    @if($exam->wasBeforeEnrollment and !isset($application))
-                    ({{ $exam->type}}, {{$exam->date->format('Y-m')}}, collegista státusz előtt szerezve)
-                    @else
-                    ({{ $exam->type}}, {{$exam->date->format('Y-m')}})
-                    @endif
-                </li>
-            @empty
-                <li>Nincs nyelvvizsga feltöltve.</li>
-            @endforelse
-        </blockquote>
+        @if ($user->educationalInformation?->languageExams()->doesntExist())
+        <blockquote>Nincs nyelvvizsga feltöltve.</blockquote>
+        @else
+            <table>
+                <thead>
+                    <tr>
+                        <th>Nyelv</th>
+                        <th>Szint</th>
+                        <th>Típus</th>
+                        <th>Dátum</th>
+                        <th></th> {{-- for the delete buttons --}}
+                    </tr>
+                </thead>
+                <tbody>
+                @foreach ($user->educationalInformation?->languageExams?->sortBy('date') ?? [] as $exam)
+                    <tr>
+                        <td><a target="_blank" href="/{{ $exam->path }}">{{ __('role.'.$exam->language) }}</a></td>
+                        <td>{{ $exam->level }}</td>
+                        <td>{{ $exam->type }}</td>
+                        <td>
+                            @if($exam->wasBeforeEnrollment and !isset($application))
+                            {{$exam->date->format('Y-m')}} (collegista státusz előtt szerezve)
+                            @else
+                            {{$exam->date->format('Y-m')}}
+                            @endif
```

Comment:
nit: now that this is a full-width table, we could maybe also include the day (the day being chopped off kind of seems like a bug)

Reply:
Right; done:)